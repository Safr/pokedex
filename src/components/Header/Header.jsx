import React, { Component } from 'react';
import { setSearchTerm } from '../../actions';
import Input from '../Input/Input';
import { Title, HeaderContainer } from './Header.style';

const Header = props => (
  <HeaderContainer>
    <Title>Pokedex</Title>
    <Input />
  </HeaderContainer>
);


export default Header;

