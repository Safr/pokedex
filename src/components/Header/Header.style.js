import styled, { css } from 'styled-components';
import media from '../../theme/media';
import { bgColor, white } from '../../theme/variables';

export const Title = styled.h1`
  margin: 0;
  margin-bottom: 10px;
  font-size: 36px;
  ${media.tablet`
   font-size: 30px;
  `}
  ${media.desktop`
    font-size: 26px;
  `}
`;

export const HeaderContainer = styled.header`
  margin: 0;
  margin-bottom: 16px;
  ${media.tablet`
    margin-bottom: 20px;
  `}
`;
