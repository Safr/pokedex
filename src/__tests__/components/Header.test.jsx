import React from 'react';
import Header from '../../components/Header/Header';

describe('Header component', () => {
  const wrapper = shallow(<Header />);
  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
