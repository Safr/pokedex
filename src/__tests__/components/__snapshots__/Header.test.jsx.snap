// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should render correctly 1`] = `
<Headerstyle__HeaderContainer>
  <Headerstyle__Title>
    Pokedex
  </Headerstyle__Title>
  <Connect(Input) />
</Headerstyle__HeaderContainer>
`;
